apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DeviceReportActivity.java

```diff
@@ -127,8 +129,12 @@ public String reportAllMicrophones() {
                     report.append(micItem);
                 }
             } catch (IOException e) {
-                e.printStackTrace();
+                Log.e(TestAudioActivity.TAG, "Caught ", e);
```

Comment:
Should we show error toast for IOExceptions?

Reply:
We did not do it before and I assume there was a reason, which I don't recall.